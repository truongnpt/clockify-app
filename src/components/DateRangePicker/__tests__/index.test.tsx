import React from 'react';
import { render } from '@testing-library/react';

import { DateRangePicker } from '..';

jest.mock('moment-timezone', () => {
  return () =>
    jest.requireActual('moment-timezone')('2021-01-01T00:00:00.000Z');
});

describe('<DateRangePicker  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<DateRangePicker onChange={jest.fn()} />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
