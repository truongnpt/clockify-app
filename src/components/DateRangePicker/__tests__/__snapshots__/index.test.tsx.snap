// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DateRangePicker  /> should match snapshot 1`] = `
<div
  class="css-1dnhiur"
>
  <div
    class="cursor-pointer form-control css-18dmyh6"
  >
    <div
      class="css-e33e47"
    />
    
    <div
      class="css-e33e47"
    />
    <span
      class="css-xndqfi"
    >
      <svg>
        chevron-down.svg
      </svg>
    </span>
  </div>
</div>
`;
